### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Oksana-24**
- [DEMO LINK](https://Oksana-24.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Oksana-24.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,101 @@
...
+  color: #00acdc;
+}
+
+li:not(:last-child) {
+  margin-right: 20px;
+}
+
+@media (min-width: 1024px) and (max-width: 1200px) {
+  a[data-qa='hover'] {
+    color: #000;
```
Comment (VitaliyBondarenko1982) [inline src/style.css:99]: redundant styles.

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,101 @@
...
+  color: #00acdc;
+}
+
+.main-nav__link:active {
+  color: #00acdc;
+}
+
+a[data-qa='hover'] {
+  color: #00acdc;
+}
```
Comment (VitaliyBondarenko1982) [inline src/style.css:91]: why do you need it? styles for hover already settled above

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Good job.
Check a couple comments, please...

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,89 @@
...
+
+.main-nav__link:hover {
+  color: #00acdc;
+}
+
+.main-nav__link:active {
+  color: #00acdc;
+}
```
Comment (etojeDenys) [inline src/style.css:85]:
```suggestion
.main-nav__link.is-active,
.main-nav__link:hover,
.main-nav__link:active {
  color: #00acdc;
}
```

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**